Makefile

```diff
@@ -69,3 +70,14 @@ e2e-bootstrap:
 .PHONY: e2e-azure
 e2e-azure:
 	bats -t test/bats/azure.bats
+
+local-e2e-bootstrap:
```

Comment:
I am not seeing any reason why e2e-bootstrap can't be reused.  What's the reasoning behind a separate target here?

Reply:
The `e2e-bootstrap` target includes the downloading of necessary tools needed for the Azure pipeline:
```bash
	# Download and install kubectl
	curl -LO https://storage.googleapis.com/kubernetes-release/release/v${KUBERNETES_VERSION}/bin/linux/amd64/kubectl && chmod +x ./kubectl && sudo mv kubectl /usr/local/bin/
	# Download and install kind
	curl -L https://github.com/kubernetes-sigs/kind/releases/download/v${KIND_VERSION}/kind-linux-amd64 --output kind && chmod +x kind && sudo mv kind /usr/local/bin/
	# Download and install Helm
	curl https://raw.githubusercontent.com/helm/helm/master/scripts/get | bash
```
Contributors will most likely have `kubectl and helm` installed on their machines, so I thought it would make more sense to create a separate `local-e2e-bootstrap` target that excludes the downloading of these binaries. 
Also, the `local-e2e-bootsrtap`  uses `Helm v3` , so I'd like to open a discussion with them on if it's possible to upgrade across the project to remove the dependency on `Tiller`

Reply:
I think a better approach would be to move the install tools to a different target so that these are re-usable.  We should open a separate issue for upgrading to Helm 3

Reply:
**The different target for the installation of the tools is a good suggestion.**

The initial thoughts around the separate target was to kick off a conversation here - the current `e2e-bootstrap` and `e2e-azure` targets are used by the Azure Pipeline, so we didn't want to break anything. 

The ultimate goal is to merge these, after a discussion and confirmation, as part of this PR.